tendrl/node_agent/manager/manager.py

```diff
@@ -99,12 +99,14 @@ def __init__(self, machine_id):
                     is None:
                 utils.delete_local_node_context()
 
+        node_id = utils.get_local_node_context()
         super(
             NodeAgentManager,
             self
         ).__init__(
             "node",
-            utils.get_local_node_context(),
+            node_id,
+            node_id,
```

Comment:
I was thinking of that. Only in case of node agent actually we need node_id and gluster/ceph integration actually dont need this. Will change the code accordingly and update the PRs.